addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,96 @@
+# Name: xpaas
+# Description: Imports xPaaS templates
+# Url: https://github.com/jboss-openshift/application-templates/tree/ose-v1.3.6
+
+oc create -f jboss-image-streams.json -n openshift
```

Comment:
I am wondering whether we should not use 'oc apply' here as @jorgemoralespou suggested. WDYT?